test/core/vector/CMakeLists.txt

```diff
@@ -29,3 +29,46 @@ sparta_named_test(Vector_test_vmulvv        Vector_test -l top info vmulvv.out
 sparta_named_test(Vector_test_vmseqvv       Vector_test -l top info vmseqvv.out       -c test_cores/test_big_core.yaml --input-file vmseqvv_e8m4.json)
 sparta_named_test(Vector_test_vsadd         Vector_test -l top info vsadd.out         -c test_cores/test_big_core.yaml --input-file vsadd.json)
 sparta_named_test(Vector_unsupported_test   Vector_test -l top info unsupported.out   -c test_cores/test_big_core.yaml --input-file vrgather.json)
+
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/elementwise.json ${CMAKE_CURRENT_BINARY_DIR}/elementwise.json SYMBOLIC)
```

Comment:
I'm confused about the changes to the tester. Some of these tests seems like they're just renamed versions of existing tests and other tests seem new. And we really only need 1 test binary. Can you merge `Vector_test.cpp` and `Vuop_test.cpp`? We only need to check the stats from the `VectorUopGenerator` and `Retire`.

Reply:
Yes, I was thinking on the Vuop_test to test the different instruction modes meanwhile the Vector_test does vector in a more general way.

I'll fuse them and make more exhaustive test with more configurations (different LMUL and SEW values)